rest-server/src/routes/job.js

```diff
@@ -40,6 +40,12 @@ router.route('/:jobName')
     /** DELETE /api/v1/jobs/:jobName - Remove job */
     .delete(tokenConfig.check, jobController.remove);
 
+router.route('/:jobName/config')
+    .get(jobController.getConfig);
```

Comment:
need authentication

Reply:
Good question. Could you explain the necessity?

Reply:
Per our discussion, we don't add authentication to this API at present.